pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -124,6 +165,15 @@ public void setUp() throws Exception {
     }
   }
 
+  @Test
+  public void testInstancesStarted() {
```

Comment:
This is because all three tests are running from within the same JVM. This will get exposed in MP, not through REST.